### ISSUE TITLE
waterproof / compact antenna?

### ISSUE THREAD

**Issue by jerabaul29** [sparkfun/SparkFun_Swarm_Satellite_Arduino_Library#12]
Hi,

That looks really exciting, looking forward to testing this product.

For real world applications, we will really need a robust, high gain, broad lobe, waterproof, antenna solution. Any ideas / tests / suggestions of kinds of antenna we could look at / try? For example, being able to use an antenna like https://www.sparkfun.com/products/16838 on the AGT is really convenient - any possibility / hope to use something similar here at some point in the future?

Also, do you think there is a possibility that in the future one could share the antenna between the satellite communications and GPS to cut costs, or is this technically impossible?

**Comment by PaulZC** [comment]
Hi JR,

Wow. The first batch of modems and boards sold out really quickly! I know we have another batch of boards in manufacture, so it won't be long before these are back in stock.

Swarm uses _much_ lower frequencies than other satellite communication systems. Swarm Tx is 148MHz to 150MHz. Just above the FM radio band in frequency. The lower frequency corresponds to long wavelengths, so the antenna needs to be physically much 'longer' as a result. Swarm have done a really good job making the antenna as compact as it is. It is quarter-wave, so there is around 50cm of wire coiled up in there!

Iridium has the 'advantage' of being much closer to the GNSS bands in frequency, making a combined antenna much easier to design.

I think there is little chance of anyone producing a 'combo' antenna. But you never know. I looked for one for the ARGOS satellite boards (401MHz) but couldn't find one...

The Swarm antenna is definitely weatherproof. It has a nice rubberized coating and the end cap is bonded on. The manual doesn't quote an IP rating, but it does say it is OK up to 100% humidity condensable. So long as you waterproof the SMA connection, you should be OK.

The [Swarm Eval Kit](https://swarm.space/product/swarm-eval-kit/) uses the solar panel as the antenna ground plane. It is very asymmetric, but it works well for them. Maybe you could do something similar?

There is more information in the modem manual about the antenna requirements (gain and VSWR). The maximum antenna gain is 5dB, to keep the modem compliant with the certification.

The manual also mentions "Swarm has reliably communicated with its satellites using a coiled ¼-wave antenna with a 50 cm long, 20 AWG (0.812mm diameter) wire descending from the ground of the SMA connector of the antenna mount". Maybe a 50cm dangling ground wire would work for you too?

I've had good results with [this antenna](https://www.sirioantenne.it/images/pdf/manuals/id-273_04-11-2019_sd2000_web.pdf).  It is very wide-band (100MHz - 2GHz). It has a gain of around 2dBi (same as the Swarm antenna). It has a VSWR of approx. 1.2 for Swarm Tx, so it's not perfect. But it is very robust, weatherproof (there is an o-ring seal between the antenna base and the mounting tube) and - from my tests - gives very good results. It does not need a separate ground plane - the 'discone' design provides the ground plane. It really isn't compact though...! Any antenna designed for the 144MHz Amateur Band will have similar performance.

Best wishes,
Paul

**Comment by jerabaul29** [comment]
Many thanks for the detailed information :) . Wonder if writing an in-depth "introduction to SWARM", comparing it a bit with older alternatives people may be better used to (Iridium / Argos) would be a good idea :) . Or maybe extending a bit the hookup guide with a paragraph about the swarm technical solution "by itself" :) .

**Comment by PaulZC** [comment]
Hi JR,

I’m hoping Swarm will publish a white paper summarising how their overall system works. I will send you a link if/when that happens. If I get some spare time, I’ll try to write my own.

A direct comparison of ‘short burst’ satellite systems is tricky because they all have their own strengths. My _personal_ take-away, _right now_, is:
* If you want the lowest message latency, choose Iridium
* If you want the lowest transmit power / current draw, choose ARGOS
* If you want the lowest message costs, choose Swarm
* (Other short burst systems are available!)

Once the Swarm constellation is complete, it really is going to be a game changer.

I’m going to close this as I believe I’ve answered your original question. As always, please open a new issue if there’s something else that needs clarifying.

Very best wishes,
Paul
